.vscode-gitpod/settings.json

```diff
@@ -1,4 +1,11 @@
 {
     "editor.tabSize": 2,
     "editor.insertSpaces": true,
+    "cmake.configureArgs": [
```

Comment:
what is this for?

Reply:
It is for the CMake plugin (our usual parameters). At first it was somehow cleaning the build folder and I had to add this to reconfigure. After deactivating the cleaning it might not be necessary since we initialize the cache at startup. But won't hurt unless we change it often.